zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/FlowNodeInstanceProcessInstanceHandler.java

```diff
@@ -89,8 +89,8 @@ public void updateEntity(
     entity.setTenantId(tenantOrDefault(recordValue.getTenantId()));
 
     // Parent tree path is intentionally not calculated here
```

Comment:
❓ now that you're setting some values below, is this comment still valid? 
🔧 if yes, could you also document more "why?"

Reply:
I made this fix as these are the values that flow node instances on very 1st level of execution (scope?) are getting. They are provided when requesting the list of flow node instances from UI. Without this change no Instance history is shown in UI. With the fix it's shown properly just without tree structure.

The comment is still valid as we will need to rework this as part of #18378.

Reply:
I will add the reference to the issue in code.